src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- <h1>Search bar airbnb</h1> -->
+    <form 
+      class="form__big"
+      action="#"
+      method="post"
+      data-qa="big"
+    >
+      <input 
+        class="form__field form__input-big"
```

Comment:
`form__input--big` modifiers are with `--` or `_`. also fix for `form__input-small`